src/AspNet.Security.OAuth.Gitter/GitterAuthenticationExtensions.cs

```diff
@@ -0,0 +1,68 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://Gitter.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using JetBrains.Annotations;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.Options;
+using System;
```

Comment:
What kind of ordering? just to know for the next provider I'll do. I'm using VS2015 and it orders it automatically for me. Do you know if I can change the order that VS2015 has by default?